test/tasks/vesting.test.ts

```diff
@@ -35,16 +35,24 @@ const MOCK_VESTING_HEADER = [
   'lastQuantityRevoked',
 ];
 
+const currentDate = new Date();
```

Comment:
were these changes still needed?

Reply:
I just wanted to update these tests so that they're more robust against time passing and had already made the changes on my branch so figured incorporate them

Reply:
I just wanted to update these tests so that they're more robust against time passing and had already made the changes on my branch so figured incorporate them